e2e/deployers/subscription.go

```diff
@@ -63,22 +76,25 @@ func (s Subscription) Undeploy(w workloads.Workload) error {
 	// Delete Subscription, Placement, Binding
 	util.Ctx.Log.Info("enter Undeploy " + w.GetName() + "/Subscription")
 
-	err := deleteSubscription()
+	name := GetCombinedName(s, w)
+	namespace := name
+
+	err := deleteSubscription(s, w)
 	if err != nil {
 		return err
 	}
 
-	err = deletePlacement()
+	err = deletePlacement(name, namespace)
 	if err != nil {
 		return err
 	}
 
-	err = deleteManagedClusterSetBinding()
+	err = deleteManagedClusterSetBinding(McsbName, namespace)
 	if err != nil {
 		return err
 	}
 
-	err = deleteNamespace()
+	err = deleteNamespace(namespace)
```

Comment:
with newly updated code, I put each application in a different namespace with following name format: deployer+workload+appname, like  subscription-deployment-busybox.  
And with cephfs sub application, it would be like subscription-deploymentfs-busybox

To test different workloads in the same namespace, that need additional field indicator the designated namespace. Will plan later.